tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -137,17 +138,36 @@ describe('package/', () => {
       await fs.promises.symlink(path.join(projectRoot, configPaths.package), hmrcLink);
     };
 
-    it('should compile without throwing an exception', async () => {
+    it('should compile with dart-sass without throwing an exception', async () => {
       await createSymlinks();
       try {
         const allScssFile = path.join(configPaths.packageTest, 'hmrc-frontend/hmrc/all.scss');
-        await sassRender({ file: allScssFile });
+        const { css } = dartSass.compile(allScssFile);
+        // TODO write a comment about why we're doing the following assertion
+        // TODO grab the bit that broke govuk-frontend before and try adding
+        //  it to our sass to see that this catches it
+        expect(css).toEqual(expect.not.stringContaining('$govuk'));
       } catch (e) {
         await removeSymLinksIfPresent();
         console.error(e.messageFormatted || e);
         throw e;
+      } finally {
+        await removeSymLinksIfPresent();
+      }
+    });
+
+    it('should compile with node-sass without throwing an exception', async () => {
+      await createSymlinks();
+      try {
+        const allScssFile = path.join(configPaths.packageTest, 'hmrc-frontend/hmrc/all.scss');
+        await nodeSassRender({ file: allScssFile });
+      } catch (e) {
+        await removeSymLinksIfPresent();
+        console.error(e.messageFormatted || e);
+        throw e;
+      } finally {
+        await removeSymLinksIfPresent();
```

Comment:
feels like we can probably tidy these tests up a bit t o make them clearer but for now I think it's ok to just duplicate stuff and have that come within some wider tech debt task to tidy up hmrc-frontend build & test config